.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Create or prepare GitHub release
+
+on:
+    push:
+        branches:    
+          - master
+        paths:
+          - 'RELEASE'
+    pull_request:
+        types: [opened, synchronize]
+        branches:
+          - master
+        paths:
+          - 'RELEASE'
```

Comment:
Will that run the release already when a pull request is created?

Reply:
I think it will trigger the action but when it checks the RELEASE file it should immediately exit since there is no "tag" entry in the file yet.

Reply:
Oops - I answered the wrong question, sorry. 

When a new PR is created after this is merged, the action will add the automatically created release notes to the PR, leaving the opportunity to make changes if desired (e.g. by using more detailed options in file `RELEASE`). The GitHub release and release tags are only created upon a PR being merged.

Reply:
More info about how this action works: 
https://github.com/pipe-cd/actions-gh-release#usage